From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marcus Brinkmann <mb@g10code.com>
Date: Thu, 17 Dec 2009 17:36:20 +0000
Subject: [PATCH] configure.ac: Make largefile check more robust

2009-12-17 Marcus Brinkmann <marcus@g10code.de>

	* configure.ac: Make largefile check more robust.
---
 ChangeLog    | 4 ++++
 configure.ac | 4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 6be60e4f..0b5c324c 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2009-12-17  Marcus Brinkmann  <marcus@g10code.de>
+
+	* configure.ac: Make largefile check more robust.
+
 2009-06-18  Marcus Brinkmann  <marcus@g10code.de>
 
 	* configure.ac (my_version): Set to 1.2.0.
diff --git a/configure.ac b/configure.ac
index 06cbb865..5bc222d0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -201,8 +201,8 @@ AC_TYPE_OFF_T
 # A simple compile time check in gpgme.h for GNU/Linux systems that
 # prevents a file offset bits mismatch between gpgme and the application.
 NEED__FILE_OFFSET_BITS=0
-case $ac_cv_sys_file_offset_bits in
-  no | unknown) ;;
+case "$ac_cv_sys_file_offset_bits" in
+  "" | no | unknown) ;;
   *)
   NEED__FILE_OFFSET_BITS=$ac_cv_sys_file_offset_bits
   ;;
